ipn/store/stores_linux.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2022 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package store provides various implementation of ipn.StateStore.
```

Comment:
delete broken-looking package doc. (it's already in the portable non-linux file and here you have a bogus newline after)